framework/test/unit/processing/spark-cicd-pipeline.test.ts

```diff
@@ -50,20 +50,12 @@ describe('With minimal configuration, the construct', () => {
   });
 
   const template = Template.fromStack(stack);
-  // console.log(JSON.stringify(template.toJSON(), null, 2));
+  console.log(JSON.stringify(template.toJSON(), null, 2));
```

Comment:
can we clean up the code here? for the readability of the tests results if there is an error